.gitignore

```diff
@@ -0,0 +1,19 @@
+# Ruby + project files
+Gemfile.lock
+Gemfile.local
+pkg
+
+# Tests
+test/reports
+coverage
+
+# RVM
+.rvmrc
+.ruby-version
+.ruby-gemset
+
+# Editors
+tags
+.idea
+*.sw?
```

Comment:
+1,000,000,000 - I kept committing .swp's the other day when I was working on the API. Can you send the same to `foreman_salt`?!